pkg/agent/controller/networkpolicy/fqdn.go

```diff
@@ -655,19 +696,18 @@ func (f *fqdnController) lookupIP(ctx context.Context, fqdn string) error {
 		}
 		return responseIPs
 	}
-
-	if f.ipv4Enabled {
+    if (scope == dnsScopeIPv4 || scope == dnsScopeDualStack) && f.ipv4Enabled {
```

Comment:
Looks like indentation is not aligned 👀

Reply:
This has already been fixed by the commit `Add UT for the fix`.

Reply:
Missed that, 👌